### PR TITLE
Alberta admin updates

### LINKED ISSUE #1044: Spruce Gove, Alberta needs label centroid and geometry
This came in through support (see Desk 1283):

Spruce Grove in Alberta is missing a label centroid and geometry, so is not returned as the locality. The user is using the WoF API, and it also reproducible with the Search API.

https://whosonfirst.mapzen.com/spelunker/id/890457029/ 

https://mapzen.com/search/explorer/?query=reverse&point.lat=53.545481&point.lon=-113.880929

(closest result is "69 Langley Crescent, AB, Canada" -- without Spruce Grove)

```
We are using whoisonfirst to generate Bread Crumbs for locations 
we have an address 51 Langley Crescent, spruce Grove or 53.545481, -113.880929 and we are querying getHierarchiesByLatLon but its not returning that this property is in Spruce Grove 

We are only getting [ 
{ 
"wof:id": 102191575, 
"wof:parent_id": "-1", 
"wof:name": "North America", 
"wof:placetype": "continent", 
"wof:country": "", 
"wof:repo": "whosonfirst-data" 
}, 
{ 
"wof:id": 85633041, 
"wof:parent_id": "102191575", 
"wof:name": "Canada", 
"wof:placetype": "country", 
"wof:country": "CA", 
"wof:repo": "whosonfirst-data" 
}, 
{ 
"wof:id": 85682091, 
"wof:parent_id": "85633041", 
"wof:name": "Alberta", 
"wof:placetype": "region", 
"wof:country": "CA", 
"wof:repo": "whosonfirst-data" 
} 
] 
```

### PR DESCRIPTION
**Pull request by stepps00**
Fixes #1044.

To do:

- [X] Figure out county level admin around Edmonton and Calgary
- [x] Ensure each new record has a `name:eng_x_preferred` property
- [x] Take "county" out of any `wof:name`
- [x] Save any `label:*` and `wof:placetype_local` property values
  - [x] municipal district, etc = county
- [x] Import/update locality records

### CODE REVIEW

File: data/112/589/442/3/1125894423.geojson [hunk 1]
```diff
@@ -33,8 +47,11 @@
     "qs_pg:pop_sr":4,
     "qs_pg:qs_id":1041613,
     "qs_pg:qs_pg_placetype":"locality",
-    "src:geom":"qs_pg",
-    "src:lbl:centroid":"qs_pg",
+    "src:geom":"can-abog",
+    "src:geom_alt":[
+        "qs_pg"
```
Comment (nvkelso) [inline data/112/589/442/3/1125894423.geojson]: Should this be `unknown`, else the alt file needs to indicate `qs_pg`?

File: data/115/886/830/9/1158868309.geojson [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    "edtf:cessation":"uuuu",
+    "edtf:inception":"uuuu",
+    "geom:area":0.000123,
+    "geom:area_square_m":928365.786057,
+    "geom:bbox":"-114.140798491,52.3478799033,-114.097409716,52.3701115648",
+    "geom:latitude":52.36012,
+    "geom:longitude":-114.115626,
+    "iso:country":"CA",
+    "label:eng_x_preferred":[
+        "S.V. of Birchcliff"
```
Comment (nvkelso) [inline data/115/886/830/9/1158868309.geojson]: Not sure why this is here... if it should be here, shouldn't it match the `name:eng_x_prefered`, but then if it's not different should be be included?

File: data/115/886/830/9/1158868309.geojson [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    "wof:breaches":[],
+    "wof:country":"CA",
+    "wof:created":1512692230,
+    "wof:geomhash":"990e6ed4403318cb092b3dd0c09ae1b3",
+    "wof:hierarchy":[],
+    "wof:id":1158868309,
+    "wof:lastmodified":1512692230,
+    "wof:name":"Birchcliff",
+    "wof:placetype":"locality",
+    "wof:placetype_local":"summer village",
```
Comment (nvkelso) [inline data/115/886/830/9/1158868309.geojson]: Good! I was wondering what SV meant.

File: data/115/886/831/3/1158868313.geojson [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    "lbl:longitude":-114.342022,
+    "mps:latitude":53.709057,
+    "mps:longitude":-114.342022,
+    "mz:hierarchy_label":1,
+    "mz:is_current":1,
+    "name:eng_x_preferred":[
+        "Castle Island"
+    ],
+    "name:eng_x_variant":[
+        "S.V. of Castle Island"
```
Comment (nvkelso) [inline data/115/886/831/3/1158868313.geojson]: We should probably add a variant that spells out `Summer Village...` to this and other summer villages.

File: data/890/456/151/890456151.geojson [hunk 5]
```diff
@@ -31,6 +44,11 @@
     "qs:photos_sr":"3",
     "qs:placetype":"Town",
     "qs:pop_sr":"3",
+    "src:geom":"can-abog",
+    "src:geom_alt":[
+        "unknown"
```
Comment (nvkelso) [inline data/890/456/151/890456151.geojson]: This geom_alt is correctly `unknown`, so not sure what's going on with the first comment.

File: data/112/589/442/3/1125894423.geojson (hunk 1 above)
Comment (stepps00) [reply data/112/589/442/3/1125894423.geojson]: Three qs_pg alt geom records have incorrect sources in their filename. Fixed in commit d83bd06735e4d.

File: data/115/886/830/9/1158868309.geojson (hunk 2 above)
Comment (stepps00) [reply data/115/886/830/9/1158868309.geojson]: Hmm.. looking at https://github.com/whosonfirst/whosonfirst-properties/blob/master/properties/label/%7Blang%7D_x_preferred.json, we should expect to see:

```
Summer Village of Birchcliff, Alberta, Canada
```

File: data/890/456/151/890456151.geojson (hunk 5 above)
Comment (stepps00) [reply data/890/456/151/890456151.geojson]: This is correct, see above comment for the qs_pg alt geom issue.

File: data/115/886/831/3/1158868313.geojson (hunk 4 above)
Comment (stepps00) [reply data/115/886/831/3/1158868313.geojson]: Will update in a new commit...

**Pull request closed without merging**